datasets/multidoc2dial/README.md

```diff
@@ -0,0 +1,465 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+- expert-generated
+languages:
+- en-US
+licenses:
+- apache-2.0
+multilinguality:
+- monolingual
+pretty_name: MultiDoc2Dial
+size_categories:
+- unknown
+source_datasets:
+- extended|doc2dial
+task_categories:
+- question-answering
+task_ids:
+- open-domain-qa
+---
+
+# Dataset Card for MultiDoc2Dial
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://doc2dial.github.io/multidoc2dial/
+- **Repository:** https://github.com/IBM/multidoc2dial
+- **Paper:** https://arxiv.org/pdf/2109.12595.pdf
+- **Leaderboard:**
+- **Point of Contact:** sngfng@gmail.com
+
+### Dataset Summary
+
+MultiDoc2Dial is a new task and dataset on modeling goal-oriented dialogues grounded in multiple documents. 
+Most previous works treat document-grounded dialogue modeling as a machine reading comprehension task based on a
+single given document or passage. We aim to address more realistic scenarios where a goal-oriented information-seeking
+conversation involves multiple topics, and hence is grounded on different documents. 
+
+### Supported Tasks and Leaderboards
+
+> Supported Task: 
+
+> Leaderboard:
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+Sample data instance for `dialogue_domain` :
+
```

Comment:
Same here: we expect to see an example from the processed dataset, not from the raw JSON data.

You can get one example by using `load_dataset` and getting the first sample `dataset["train"][0]`